pkg/common/adaptor/request.go

```diff
@@ -38,3 +38,19 @@ func GetCompatRequest(req *protocol.Request) (*http.Request, error) {
 	r.Header = h
 	return r, nil
 }
+
+func SwapToHertzRequest(req *http.Request, hreq *protocol.Request) error {
```

Comment:
Here the function name is changed to "CopyTo" which seems to be more semantic

Reply:
done